mod/multi_file_upload/actions/file/upload.php

```diff
@@ -66,108 +60,113 @@
 	$title = htmlspecialchars($_FILES['upload']['name'][$i], ENT_QUOTES, 'UTF-8');
 	$title2 = $title;
 
+	$file->title = $title;
+	$file->title2 = $title2;
+	$file->title3 = gc_implode_translation($title,$title2);
+	$file->description = $desc;
+	$file->description2 = $desc2;
+	$file->description3 = gc_implode_translation($desc,$desc2);
+	$file->access_id = $access_id;
+	$file->container_guid = $container_guid;
+	//$file->tags = string_to_tag_array($tags);
```

Comment:
I have the habit of correcting indentations, for sake of readability (for me anyways)